src/js/providers/html5.js

```diff
@@ -11,8 +11,8 @@ define([
 ], function(cssUtils, utils, stretchUtils, _, events, states, eventdispatcher, DefaultProvider, video) {
 
     var clearInterval = window.clearInterval,
-        stallInterval,
         STALL_DELAY = 256,
+        BUFFER_INTERVAL = 50,
```

Comment:
I think more buffer updates are good, and we're more likely to hit the stall delay earlier (might be better if we make stall delay a little less than a multiple or buffer interval e.g. 240)